cypress/support/fragments/data_import/match_profiles/matchProfileView.js

```diff
@@ -0,0 +1,24 @@
+import { HTML, including } from '@interactors/html';
+import {
+  Button,
+  Pane,
+  Accordion
+} from '../../../../../interactors';
+
+const viewPane = Pane({ id:'view-match-profile-pane' });
+
+export default {
+  edit:() => {
+    cy.wait(1500);
```

Comment:
add comment for reason of this waiter pls

Reply:
Done